src/CACluster.cc

```diff
@@ -259,6 +261,11 @@ void CACluster::clusterProperties()
       // CSCs -> take only one Z value per each station
       
       if(rechit.superlayer > 0){ //hit in DT
+	//save nhits per wheel
+	if(abs(rechit.z) < 126.8){tmpCluster.nDtRechitsWheel0++;}
```

Comment:
There's this variable in the big ntuple that's called `dtRechitWheel`, can you use that to assign the wheel instead of Z?

Reply:
looks like the existing branch stores the wheel which contains the biggest number of hits for each cluster. The new variables I added correspond to the number of hits found in each wheel. 

Several clusters have hits in more than one one wheel:

![image](https://user-images.githubusercontent.com/22025854/233001963-bffafcb7-bc0a-4c32-b872-1cbc3aaa0048.png)

Reply:
Ok, I got confused with what "the big ntuple" meant. I'm updating that

Reply:
Done